diff --git a/signals.py b/signals.py
index 307c8ce..2ef2b33 100644
--- a/signals.py
+++ b/signals.py
@@ -3,7 +3,7 @@ from __future__ import unicode_literals
 import django.dispatch
 
 
-user_signed_up = django.dispatch.Signal(providing_args=["user", "form"])
+user_signed_up = django.dispatch.Signal(providing_args=["user", "form", "request"])
 user_sign_up_attempt = django.dispatch.Signal(providing_args=["username", "email", "result"])
 user_logged_in = django.dispatch.Signal(providing_args=["user", "form"])
 user_login_attempt = django.dispatch.Signal(providing_args=["username", "result"])
diff --git a/views.py b/views.py
index 6c7189c..4b27e0e 100644
--- a/views.py
+++ b/views.py
@@ -212,7 +212,7 @@ class SignupView(FormView):
         email_address.send_confirmation(site=get_current_site(self.request))
 
     def after_signup(self, form):
-        signals.user_signed_up.send(sender=SignupForm, user=self.created_user, form=form)
+        signals.user_signed_up.send(sender=SignupForm, user=self.created_user, form=form, request=self.request)
 
     def login_user(self):
         user = self.created_user
