diff --git a/signals.py b/signals.py
index 2ef2b33..d4a56bd 100644
--- a/signals.py
+++ b/signals.py
@@ -12,3 +12,4 @@ signup_code_used = django.dispatch.Signal(providing_args=["signup_code_result"])
 email_confirmed = django.dispatch.Signal(providing_args=["email_address"])
 email_confirmation_sent = django.dispatch.Signal(providing_args=["confirmation"])
 password_changed = django.dispatch.Signal(providing_args=["user"])
+updated_account = django.dispatch.Signal(providing_args=["user", "request"])
diff --git a/views.py b/views.py
index 4b27e0e..4a2a888 100644
--- a/views.py
+++ b/views.py
@@ -713,6 +713,10 @@ class SettingsView(LoginRequiredMixin, FormView):
     def update_settings(self, form):
         self.update_email(form)
         self.update_account(form)
+        self.after_update_account()
+        
+    def after_update_account(self):
+        signals.updated_account.send(sender=SettingsView, user=self.request.user, request=self.request)
 
     def update_email(self, form, confirm=None):
         user = self.request.user
